src/views/docs-view/utils/get-deploy-preview-loader.ts

```diff
@@ -61,16 +62,17 @@ export function getDeployPreviewLoader({
 	return new FileSystemLoader({
 		...loaderOptions,
 		...fsOptions,
-		remarkPlugins(params) {
+		remarkPlugins(params, version) {
 			const remarkPluginsFromExtraOptions =
 				typeof loaderOptions.remarkPlugins === 'function'
 					? loaderOptions.remarkPlugins(params)
 					: loaderOptions.remarkPlugins
 
-			// Custom handling for TF projects whose images reside in `website/img`
-			// rather than `website/public` (Next.js serves static assets from `public`)
+			// We have some custom handling for TF projects who have a few project-level variances
+			// - images reside in `website/img` rather than `website/public` (Next.js serves static assets from `public`)
+			// - TFE/C content exclusion markdown comments are used
 			const remarkTerraformPlugins = []
-			if (currentRootDocsPath.productSlugForLoader?.includes('terraform')) {
+			if (currentRootDocsPath.productSlugForLoader?.match(/(^terraform-)/i)) {
```

Comment:
Should this apply to content from `hashicorp/terraform` as well?

Reply:
yes it should!

Reply:
Fixed